cypress/e2e/immutableDatabaseTests/linkCheck.ts

```diff
@@ -0,0 +1,104 @@
+import { setTokenUserViewPort } from '../../support/commands';
+
+function formatPaths(paths: string[]) {
+  let formattedPaths = '';
+  paths.forEach((path) => {
+    formattedPaths += '\n- ' + path;
+  });
+  return formattedPaths;
+}
+
+describe('Find broken anchor links', () => {
+  setTokenUserViewPort();
+
+  function checkUrls(path: string, selector: string) {
+    let brokenUrls = [];
+    cy.visit(path);
+    cy.get(selector)
+      .find('a')
+      .each((url) => {
+        cy.get(url).then((url) => {
+          const href = url.prop('href');
+          if (href) {
+            cy.request({
+              url: href,
+              failOnStatusCode: false,
+            }).then((result) => {
+              if (result.status != 200) {
+                brokenUrls.push(href);
+                cy.log(result.status + ': ' + href);
+              }
+            });
+          }
+        });
+      })
+      .then(() => {
+        if (brokenUrls.length) {
+          throw new Error(`Broken links at "${path}":` + formatPaths(brokenUrls));
+        }
+      });
+  }
+
+  const urlPages = [
```

Comment:
Could make this a typed array, that way you could refer to the properties by name (something like "path" and "selector") rather than by array index.